### ISSUE TITLE
LatexTools stopped working on windows 10 after the latest update.

### ISSUE THREAD

**Issue by judios** [SublimeText/LaTeXTools#930]
After the update on November 6 2016, LatexTools stopped working in two of my computers with windows 10.

The message I got is 

```
TraditionalBuilder: Engine: pdflatex. Invoking latexmk... 

COULD NOT COMPILE!

Attempted command:latexmk -cd -f -pdf -interaction=nonstopmode -synctex=1 OverView.tex
Build engine: Traditional Builder
```

**Comment by r-stein** [comment]
Have you tried to restart ST? Do you see an error when opening the ST console ``ctrl+` `` when the build fails?

**Comment by judios** [comment]
It seems that the problem is with the way the "texpath" variable is handled. 

Since I have a portable version of texlive, I didn't have the bin folder of texlive in the system path. Still, I have set the textpath variable in the  prefferences user file. Up to the last version it worked fine.

What I did to fix the problem, I added the texlive bin folder to the system path.

**Comment by ig0774** [comment]
@judios: Can you post what your `texpath` settings entry looks like? It should work...

**Comment by judios** [comment]
`"windows": {
		// Path used when invoking tex & friends; "" is fine for MiKTeX
		// For TeXlive 2011 (or other years) use
		// "texpath" : "C:\\texlive\\2011\\bin\\win32;$PATH",
		"texpath" : "D:\\texlive\\bin\\win32;$PATH",
		// TeX distro: "miktex" or "texlive"
		"distro" : "texlive",
		// Command to invoke Sumatra. If blank, "SumatraPDF.exe" is used (it has to be on your PATH)
		"sumatra": "D:\\Software\\Sumatra\\SumatraPDF.exe",
		// Command to invoke Sublime Text. Used if the keep_focus toggle is true.
		// If blank, "subl.exe" or "sublime_text.exe" will be used.
		"sublime_executable": "",
		// how long (in seconds) to wait after the jump_to_pdf command completes
		// before switching focus back to Sublime Text. This may need to be
		// adjusted depending on your machine and configuration.
		"keep_focus_delay": 0.5
	},`

**Comment by CremboC** [comment]
I'm getting the same on macOS. Also, checking the console (ctrl+`) and I get:
```
    <removed>/Packages/LaTeXTools/builders/basicBuilder.py", line 237, in run_bibtex
    preexec_fn=os.setid if sublime.platform() != 'windows' else None,
AttributeError: 'module' object has no attribute 'setid'
```

**Comment by ig0774** [comment]
@CremboC: That's actually a completely different error. Would you mind opening a separate issue for it?

**Comment by judios** [comment]
I have tracked the bug to this commit

Adds a standard way to run external commands. on 10/30/2016 by Ian

**Comment by CremboC** [comment]
@ig0774 the error that I get is actually
```
Basic Builder: running pdflatex...done.


COULD NOT COMPILE!


Build engine: Basic Builder
```

Only when I open the console I see those details. Are you sure it's a different error?

**Comment by ig0774** [comment]
Yes. The initial error is due to the `texpath` not being passed correctly on Windows 10. The error you're seeing is due to a typo in the part of the `basic` builder that runs `bibtex`.

**Comment by judios** [comment]
This the error I get in the console window

`<class '_latextools_traditionalBuilder.TraditionalBuilder'>
2
Welcome to thread Thread-4
['latexmk', '-cd', '-f', '-pdf', '-interaction=nonstopmode', '-synctex=1', 'OverView.tex']
Running "latexmk -cd -f -pdf -interaction=nonstopmode -synctex=1 OverView.tex"
Traceback (most recent call last):
  File "./python3.3/subprocess.py", line 1104, in _execute_child
FileNotFoundError: [WinError 2] The system cannot find the file specified

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\jdiaz\AppData\Roaming\Sublime Text 3\Packages\LaTeXTools\makePDF.py", line 293, in run
    cwd=self.caller.tex_dir
  File "C:\Users\jdiaz\AppData\Roaming\Sublime Text 3\Packages\LaTeXTools\latextools_utils\external_command.py", line 190, in external_command
    cwd=cwd
  File "./python3.3/subprocess.py", line 819, in __init__
  File "./python3.3/subprocess.py", line 1110, in _execute_child
FileNotFoundError: [WinError 2] The system cannot find the file specified`

**Comment by ig0774** [comment]
@judios: Thanks for the bug report. I apparently wasn't looking to closely at the behaviour on Windows 10. It should be fixed in v3.12.7.

**Comment by judios** [comment]
@ig0774  Thank you for your fast response to this issue.

I have tested your solution and it works perfect.

Thanks
